app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,10 +1,39 @@
 import React from "react";
 
+import * as yup from "yup";
+
 import { convertDateToStdFormat } from "utils/dayjs 4.31.02 PM";
 
 import Avatar from "./Table/Avatar";
 import RowActions from "./Table/RowActions";
 
+// Start of Form component constants. --------------------------------
```

Comment:
Please remove unnecessary comments like these. It doesn't add any particular value at the moment.